fiorella/src/snippets/pagination.liquid

```diff
@@ -1,12 +1,32 @@
-<div class="pagination">
+<div class="pagination pagination-controls">
   {% if paginate.previous.is_link %}
-    <a href="{{ paginate.previous.url }}" class="prev-button">« PREV</a>
+  <a href="{{ paginate.previous.url }}" class="prev-button">
```

Comment:
This code can be simplified by creating a variable using liquid, one for the `url` that is going to be inside the `href` of the anchor, and another one for the `class` of the anchor as well... it's just a suggestion

Reply:
Actually, this code can be simplified to be just a small piece of HTML... as I said, it depends on you if you keep as it is right now, if you want to simplify the code then go ahead 🤓